web/templates/attendance/index_coordinator.html.eex

```diff
@@ -32,8 +32,18 @@
             >
               <i class="material-icons">more_vert</i>
             </button>
-            <ul class="mdl-menu mdl-menu--bottom-right mdl-js-menu"
-                for="tr_menu_<%= offered_course.id %>"
+            <ul
+              class="
+                mdl-menu mdl-js-menu
+                <%=
+                  if index > 10 && index > length(@offered_courses)-4 do
```

Comment:
The pretty solution would be to create a separated reusable list component, that we would use instead of copying the same html to everywhere, but that's not the case now. I can extract the condition, but I don't think that would help so much.